source/fundamentals/crud/read-operations/cursor.txt

```diff
@@ -78,6 +84,43 @@ closes if there is an early termination:
 
 For more information on how to ensure a cursor closes, see the :ref:`cursor cleanup section <cursor_cleanup>`.
 
+Explain
+~~~~~~~
+
+Use the :java-docs:`explain() <apidocs/mongodb-driver-sync/com/mongodb/client/FindIterable.html#explain()>`
+method to explain the execution plan of the operation with the given
+:java-core-api:`verbosity level <com/mongodb/ExplainVerbosity>`: 
+
+.. literalinclude:: /includes/fundamentals/code-snippets/Cursor.java
+   :language: java
+   :dedent:
+   :start-after: begin explainExample
+   :end-before: end explainExample
+
+The following table shows the verbosity levels you can give an
+operation, what they do, and what their intended use case is:
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+   :widths: 30 40 30
+
+   * - Verbosity Level
+     - Description
+     - Use Case
+
+   * - ALL_PLANS_EXECUTIONS
+     - Runs the query optimizer and chooses the winning plan, but then runs all generated plans to completion.
+     - Which plan will MongoDB choose to run my query?
+
+   * - EXECUTION_STATS
+     - Runs the query optimizer, and then runs the winning plan to completion.
+     - Is my query performing well?
+
+   * - QUERY_PLANNER
+     - Runs the query planner and chooses the winning plan, but does not actually execute it.
+     - I have a problem with this query, and I want as much information as possible to diagnose why it might be slow.
+
```

Comment:
Suggestion: I think this table is identical to the table on the aggregation page. Could both tables be sourced from an included file? Having two separate tables intended to be identical raises maintenance concerns.